internal/cmd/trust/trust.go

```diff
@@ -47,6 +48,7 @@ func New() *cobra.Command {
 	cmd.AddCommand(apply.New())
 	cmd.AddCommand(disablegithubappapprovals.New(o))
 	cmd.AddCommand(enablegithubappapprovals.New(o))
+	cmd.AddCommand(hooks.New(o))
```

Comment:
for consistency, this should probably be `gittuf trust add-hook` etc instead of `gittuf trust hooks ...`.